apps/storefront/package.json

```diff
@@ -23,13 +23,19 @@
     "@graphql-codegen/typescript-apollo-client-helpers": "2.2.5",
     "@headlessui/react": "1.7.3",
     "@heroicons/react": "1.0.6",
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/icons": "^4.11.3",
+    "@material-ui/lab": "4.0.0-alpha.61",
+    "@saleor/app-sdk": "^0.18.0",
+    "@saleor/macaw-ui": "^0.6.6",
```

Comment:
Macaw is used to build this single page in dashboard, but it requires material to be installed.

Its Next app so I believe there should be no bundle impact on public code, but not sure. Also Macaw/Material can introduce some problems with peers.

Since we only need very simple page in dashboard (1-field form) I can make this page without Macaw. It may not be 100% consistent, but maybe its be better to drop this legacy.

WDYT @mmiszy ?